reconcile/cli.py

```diff
@@ -219,6 +220,14 @@ def init_log_level(log_level):
     logging.basicConfig(format=format,
                         datefmt='%Y-%m-%d %H:%M:%S',
                         level=level)
+    if 'syslog' in config.get_config():
+        syslog_server = config.get_config()['syslog']['server']
+        syslog_port = int(config.get_config()['syslog']['port'])
+        syslog = SysLogHandler(address=(syslog_server, syslog_port))
+        formatter = logging.Formatter(format, datefmt='%Y-%m-%d %H:%M:%S')
```

Comment:
can we extract the date format to a variable and re-use it?

Reply:
also, does this mean we are shipping logs from all levels?

Reply:
@BumbleFeng @maorfr Raised a good point. We should make log level for CloudWatch configurable. Debug logs may contain messages that should not be stored in external systems. We should make sure only log messages at `INFO` level or above are shipped to CloudWatch.

Reply:
`syslog.setLevel(log_level)` should work